src/redux/api/slice.js

```diff
@@ -108,23 +109,24 @@ export const apiSlice = createApi({
           '/api/users',
           100,
           {
-            'roles[]': 'ROLE_COURIER'
-          }
+            'roles[]': 'ROLE_COURIER',
+          },
         );
 
-        return result ? { data: sortByString(result, 'username') } : { error: "result.error" };
+        if (result.error) {
```

Comment:
I still prefer the 1line ternary `if` with only 1 `return`

Reply:
I applied the latest prettier style, let's stick to it